src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -1,10 +1,5 @@
 <Project>
 
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <ProjectCapability Include="UseMauiCore" />
-    <ProjectCapability Condition=" '$(UseMaui)' == 'true' " Include="UseMaui" />
-  </ItemGroup>
```

Comment:
Do we lose the `UseMaui` capability? Lutz heavily uses this with VS's telemetry.

`UseMauiCore` is also nice if someone wants to find out usage about that.